test/date-time/date-time-create.test.ts

```diff
@@ -0,0 +1,1068 @@
+import * as Assert from "assert";
+import { DateTime } from "../../src/date-time";
+import { DateTime as LuxonDateTime } from "luxon";
+
+
+suite("DateTime Create", () =>
+{
+    suite("Constructor", () =>
+    {
+        const validValue = "2024-01-01 10:00";
+
+        function checkIsInvalid(value: string, zone = "utc"): void
+        {
+            let dateTime = "";
+            try
+            {
+                dateTime = new DateTime({ value, zone }).toString();
+            }
+            catch (e: any)
+            {
+                // console.log(e.reason);
+                Assert.ok(e.reason);
```

Comment:
umm not sure about this, Asset.ok take in a boolean, what if the reason was an empty string it would fail this